test/data/css/tests.js

```diff
@@ -304,7 +304,7 @@ exports.test_data = {
                 { unchanged: '/* test */' },
                 { input: '.tabs{/* test */}', output: '.tabs {\n\t/* test */\n}' },
                 { input: '.tabs{/* test */}', output: '.tabs {\n\t/* test */\n}' },
-                { input: '/* header */.tabs {}', output: '/* header */\n\n.tabs {}' },
+                { input: '/* header */.tabs {}', output: '/* header */\n.tabs {}' },
```

Comment:
It doesn't seem like this behavior should change. 
I think the header followed by a blank line should be maintained.

Reply:
@amandabot 
Whoa.  Okay, let me look at this a bit more.